thunder/core/jit_ext.py

```diff
@@ -791,7 +791,22 @@ def _generate_random_str_id() -> str:
     # note that this key is quite new: https://github.com/pytorch/pytorch/pull/134087
     # non_differentiable_idx = fwd_kwargs.get("non_differentiable_idx")
     length_of_tensor_args = sum(args_tensor_mask)
-    new_fwd_args = (wrap_const(None),) + fwd_args[:length_of_tensor_args]
+
+    # N.B.(crcrpar) When `torch.compile(..., dynamic=True)`,
+    # GraphModules' forward seem to take `SymInt` and other values
+    # as its argument with some probability. Though that piece of information unfortunately
+    # does not seem to be indicated in ``args_tensor_`` nor ``non_differentiable_idx``.
```

Comment:
I think I wrote this line but I guess this is a typo of
```suggestion
    # does not seem to be indicated in ``args_tensor_mask`` nor ``non_differentiable_idx``.
```